[2024-09-01T13:10:07.159+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: k9_etl_dag.load manual__2024-09-01T13:10:01.146116+00:00 [queued]>
[2024-09-01T13:10:07.165+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: k9_etl_dag.load manual__2024-09-01T13:10:01.146116+00:00 [queued]>
[2024-09-01T13:10:07.166+0000] {taskinstance.py:1359} INFO - Starting attempt 1 of 1
[2024-09-01T13:10:07.172+0000] {taskinstance.py:1380} INFO - Executing <Task(_PythonDecoratedOperator): load> on 2024-09-01 13:10:01.146116+00:00
[2024-09-01T13:10:07.181+0000] {standard_task_runner.py:57} INFO - Started process 82 to run task
[2024-09-01T13:10:07.184+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'k9_etl_dag', 'load', 'manual__2024-09-01T13:10:01.146116+00:00', '--job-id', '31', '--raw', '--subdir', 'DAGS_FOLDER/k9_etl_dag.py', '--cfg-path', '/tmp/tmprym_2z7_']
[2024-09-01T13:10:07.185+0000] {standard_task_runner.py:85} INFO - Job 31: Subtask load
[2024-09-01T13:10:07.246+0000] {task_command.py:415} INFO - Running <TaskInstance: k9_etl_dag.load manual__2024-09-01T13:10:01.146116+00:00 [running]> on host 68d4b554a795
[2024-09-01T13:10:07.317+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='k9_etl_dag' AIRFLOW_CTX_TASK_ID='load' AIRFLOW_CTX_EXECUTION_DATE='2024-09-01T13:10:01.146116+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-09-01T13:10:01.146116+00:00'
[2024-09-01T13:10:07.319+0000] {logging_mixin.py:151} INFO - Data loaded into /opt/***/data/k9_output.txt
[2024-09-01T13:10:07.319+0000] {python.py:194} INFO - Done. Returned value was: None
[2024-09-01T13:10:07.323+0000] {taskinstance.py:1398} INFO - Marking task as SUCCESS. dag_id=k9_etl_dag, task_id=load, execution_date=20240901T131001, start_date=20240901T131007, end_date=20240901T131007
[2024-09-01T13:10:07.374+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2024-09-01T13:10:07.387+0000] {taskinstance.py:2776} INFO - 0 downstream tasks scheduled from follow-on schedule check
